### ISSUE TITLE
Error: Failed to convert Strength/Stretching Workouts

### ISSUE THREAD

**Issue by bombbr** [philosowaffle/peloton-to-garmin#138]
**Describe the bug**
Was having an issue with the 404 error that was just fixed (https://github.com/philosowaffle/peloton-to-garmin/issues/136) but now the following error occurs when running program:

I've replaced the workout id with WORKOUTID and my username

[18:47:20 ERR] Failed to convert workout data WORKOUTID_30_min_Focus_Flow-_Hamstrings_with_Ross_Rayburn ./working/downloaded/WORKOUTID_30_min_Focus_Flow-_Hamstrings_with_Ross_Rayburn.json
System.InvalidOperationException: Nullable object must have a value.
   at System.Nullable`1.get_Value()
   at Conversion.FitConverter.GetSessionMesg(Workout workout, WorkoutSamples workoutSamples, DateTime startTime, DateTime endTime, UInt16 numLaps) in /Users/xxx/peloton-to-garmin/src/Conversion/FitConverter.cs:line 424
   at Conversion.FitConverter.Convert(Workout workout, WorkoutSamples workoutSamples) in /Users/xxx/peloton-to-garmin/src/Conversion/FitConverter.cs:line 150
   at Conversion.Converter`1.Convert(String format) in /Users/xxx/peloton-to-garmin/src/Conversion/IConverter.cs:line 94

**Installation (please complete the following information):**
 - OS: mac
 - Method: source
 - Version: 2.0.3

**Comment by Novelus23** [comment]
I am getting the same item with a strength workout, cycling appears fine: [22:12:39 ERR] Failed to convert workout data cccc_20_min_Upper_Body_Strength_with_Adrian_Williams ./working\downloaded\cccc_20_min_Upper_Body_Strength_with_Adrian_Williams.json
System.InvalidOperationException: Nullable object must have a value.
   at System.Nullable`1.get_Value()
   at Conversion.FitConverter.GetSessionMesg(Workout workout, WorkoutSamples workoutSamples, DateTime startTime, DateTime endTime, UInt16 numLaps) in D:\a\peloton-to-garmin\peloton-to-garmin\src\Conversion\FitConverter.cs:line 424
   at Conversion.FitConverter.Convert(Workout workout, WorkoutSamples workoutSamples) in D:\a\peloton-to-garmin\peloton-to-garmin\src\Conversion\FitConverter.cs:line 148
   at Conversion.Converter`1.Convert(String format) in D:\a\peloton-to-garmin\peloton-to-garmin\src\Conversion\IConverter.cs:line 90

**Comment by philosowaffle** [comment]
Just pushed a [commit](https://github.com/philosowaffle/peloton-to-garmin/commit/207fbe34b6a8f4c3eb54e6d83ca40fd693aa9314) straight to master that attempts to fix this. Pull latest and give it a go. If that doesn't work I'll try to do a stretching or strength workout tomorrow so I have data to reproduce with.

**Comment by bombbr** [comment]
Worked on my end, thank you!

**Comment by Novelus23** [comment]
@philosowaffle my apologies, I'm a windows exe user, how would I go about updating my application with the latest?

**Comment by philosowaffle** [comment]
@Novelus23 - I updated the docs to include instructions on how to [get the latest exe](https://github.com/philosowaffle/peloton-to-garmin/wiki/Quick-Start#latest-executable), looks like they were previously a little out of date, but should be correct now.

**Comment by philosowaffle** [comment]
Fixed with [v2.0.4](https://github.com/philosowaffle/peloton-to-garmin/releases/tag/v2.0.4)

**Comment by iggbom** [comment]
I got this same issue with a bike workout with code that I pulled after the #136 fix. I don't think its specific to stretching or strength workouts.

The difference with previous workouts is that this specific workout I was not wearing an HR monitor so no HR data was captured.

```
2021-07-13 18:02:41.531 -07:00 [ERR] Failed to convert workout data ed0afa0436704841bf890e666da1de39_30_min_Power_Zone_Endurance_Ride_with_Olivia_Amato ./working/downloaded/ed0afa0436704841bf890e666da1de39_30_min_Power_Zone_Endurance_Ride_with_Olivia_Amato.json
System.InvalidOperationException: Nullable object must have a value.
   at System.Nullable`1.get_Value()
   at Conversion.FitConverter.GetSessionMesg(Workout workout, WorkoutSamples workoutSamples, DateTime startTime, DateTime endTime, UInt16 numLaps) in /build/src/Conversion/FitConverter.cs:line 436
   at Conversion.FitConverter.Convert(Workout workout, WorkoutSamples workoutSamples) in /build/src/Conversion/FitConverter.cs:line 148
   at Conversion.Converter`1.Convert(String format) in /build/src/Conversion/IConverter.cs:line 90
```
